runtime/src/bank/partitioned_epoch_rewards/mod.rs

```diff
@@ -405,6 +406,11 @@ mod tests {
             mut genesis_config, ..
         } = create_genesis_config_with_vote_accounts(1_000_000_000, &validator_keypairs, stakes);
         genesis_config.epoch_schedule = EpochSchedule::new(SLOTS_PER_EPOCH);
+        genesis_config
+            .accounts
+            .get_mut(&epoch_rewards::id())
+            .unwrap()
+            .set_lamports(1);
```

Comment:
this seems fine to me to prevent `EpochRewards` from being immediately deallocated instead of adding back the `slot == SLOTS_PER_EPOCH` hack. these tests use `Rent::free()` because there are rent-paying accounts so using `Rent::default()` would mess up the `capitalization()` checks